### PR TITLE
Move ZapZap entry to communication apps, remove redundant zoom entry

### PR DESCRIPTION
**Pull request by jeevithakannan2**
## Type of Change
- [x] Documentation update
- [x] Refactoring

## Description
- Remove redundant zoom entry and move zapzap from developer tools to communication.

## Testing

**Before**

image
image

**After**

image
image

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: docs/userguide.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 - **Slack**: Slack is a collaboration platform designed for team communication, featuring channels, direct messaging, file sharing, and integrations with various productivity tools.
 - **Telegram**: Telegram is a cloud-based messaging app known for its speed and security, offering features like group chats, channels, and end-to-end encrypted calls.
 - **Thunderbird**: Thunderbird is a free, open-source email client that offers powerful features like customizable email management, a built-in calendar, and extensive add-ons for enhanced functionality.
+- **ZapZap**: ZapZap is an open source whatsapp desktop client for Linux users developed by rafatosta.
```
Comment (Real-MullaC) [inline docs/userguide.md:15]: Remove this and add it to the https://github.com/Chris-Titus-Docs/linutil-docs repo

**Review by Real-MullaC — CHANGES_REQUESTED** [review]
This is to make no conflicts with PR https://github.com/ChrisTitusTech/linutil/pull/888

**Review by ChrisTitusTech — CHANGES_REQUESTED** [review]
Why remove Zoom that is used all over the US for meetings.

**Comment by jeevithakannan2** [comment]
> Why remove Zoom that is used all over the US for meetings.

That is a redundant entry. It is not completely removed.

**Comment by harshv5094** [comment]
> ## Type of Change
> 
>     * [x]  Documentation update
> 
>     * [x]  Refactoring
> 
> 
> ## Description
> 
>     * Remove redundant zoom entry and move zapzap from developer tools to communication.
> 
> 
> ## Testing
> 
> **Before**
> 
> image image
> 
> **After**
> 
> image image
> ## Checklist
> 
>     * [x]  My code adheres to the coding and style guidelines of the project.
> 
>     * [x]  I have performed a self-review of my own code.
> 
>     * [x]  I have made corresponding changes to the documentation.
> 
>     * [x]  My changes generate no errors/warnings/merge conflicts.

Sorry, My bad 😝

**Comment by jeevithakannan2** [comment]
> Sorry, My bad 😝

No problem

**Review by adamperkowski — APPROVED** [review]

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**